packages/app/src/runner/event-manager.ts

```diff
@@ -404,6 +408,9 @@ export class EventManager {
 
           if (!hideCommandLog) {
             this.reporterBus.emit('runnables:ready', runnables)
```

Comment:
Looking through this review, we are splitting the total test count and testFilter on specStore, runnablesStore, etc. Have you considered returning metadata from normalizeAll on either the runnables and/or another return signature to told this information?
```js
const runData = Cypress.runner.normalizeAll(runState.tests, hideCommandLog, specStore.testFilter)

this.reporterBus.emit('runnables:ready', runData)

// reporter
const {
  runnables,
  totalTests,
  totalUnfilteredTests,
  specFilter
} = runData

// do stuff with runnables & totals & spec filter
```

Reply:
I'll see what I can do here, I agree the info is a bit spread out and would be nice to have it consolidated.